docs/docs/pricing/README.md

```diff
@@ -11,15 +11,238 @@ To support these goals, **Pipedream offers a [generous free tier](#developer-tie
 
```

Comment:
https://pipedream-docs-ltr2yeku5-pipedreamers.vercel.app/docs/pricing/#professional-tier , https://pipedream-docs-ltr2yeku5-pipedreamers.vercel.app/docs/pricing/#team-plan and https://pipedream-docs-ltr2yeku5-pipedreamers.vercel.app/docs/pricing/#enterprise-plan still have some out-of-date information